Libraries/CustomComponents/NavigationExperimental/NavigationHeader.js

```diff
@@ -133,11 +138,17 @@ class NavigationHeader extends React.Component<DefaultProps, Props, any> {
     });
 
     return (
-      <View style={[ styles.appbar, style ]} {...viewProps}>
+      <Animated.View style={[
+          styles.appbar,
+          { height: APPBAR_HEIGHT + this.props.statusBarHeight },
```

Comment:
We can't add a plain number and an `Animated.Value`, right? May be just have a simple `Animated.View` as child on which the statusbar height is set?